Runtime/Scripts/MobiledgeXIntegrationHelper.cs

```diff
@@ -131,15 +131,23 @@ private Loc GetLocationFromDevice()
         {
             // Location is ephemeral, so retrieve a new location from the platform. May return 0,0 which is
             // technically valid, though less likely real, as of writing.
+#if UNITY_EDITOR
+            Debug.Log("MobiledgeX: Cannot Get location in Unity Editor. Returning dummy location.");
+            return new Loc{
+                latitude = 37.3382082,
+                longitude = -121.8863286
```

Comment:
Oh, just getting visions of the whole (long, lat), versus (lat, long) debate. But here, they're named them at least.